src/main/java/de/doubleslash/keeptime/common/ScreenPosManager.java

```diff
@@ -0,0 +1,117 @@
+package de.doubleslash.keeptime.common;
+
+import de.doubleslash.keeptime.controller.Controller;
+import de.doubleslash.keeptime.model.Model;
+import de.doubleslash.keeptime.model.Settings;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.stage.Screen;
+import javafx.stage.Stage;
+
+public class ScreenPosManager {
+
+   Stage stage;
+   Model model;
+   Controller controller;
+
+   public ScreenPosManager(final Stage stage, final Model model, final Controller controller) {
```

Comment:
class should have just one concern, handling screen coordinates. Adding dependecies to other components makes it complex and hard to understand.
it should not be the concern of the class to e.g. update a model. Separation of Concerns
it should only be responsible to make crazy screen calculations. Single Responsibility

Other classes can use the functionaly of the class.